### PR TITLE
Circleci nightly build

### PR DESCRIPTION
**Pull request by jmleroux**
To test the correct installation with community dev.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+          name: Start containers
+          command: docker-compose up -d
+      - run:
+          name: Install dependencies
+          command: ./bin/docker/pim-dependencies.sh
+      - run:
+          name: Pim installation
+          command: |
+              docker-compose exec fpm bin/console --env=prod pim:install --force --clean
+              docker-compose run --rm node yarn run webpack
```
Comment (ahocquard) [inline .circleci/config.yml]:
```suggestion
            &&  docker-compose run --rm node yarn run webpack
```

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      - run:
+          name: Start containers
+          command: docker-compose up -d
+      - run:
+          name: Install dependencies
+          command: ./bin/docker/pim-dependencies.sh
+      - run:
+          name: Pim installation
+          command: |
+              docker-compose exec fpm bin/console --env=prod pim:install --force --clean
```
Comment (ahocquard) [inline .circleci/config.yml]:
```suggestion
              docker-compose exec fpm bin/console --env=prod pim:install --force --clean \
```

File: bin/docker/pim-setup.sh [hunk 3]
```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+set -e
+cd "$(dirname "$0")"
+cd ./../../
+
+if [ ! -f ./app/config/parameters.yml ]; then
+    cp ./app/config/parameters.yml.dist ./app/config/parameters.yml
+    sed -i "s/database_host:.*localhost/database_host: mysql/g" ./app/config/parameters.yml
```
Comment (ahocquard) [inline bin/docker/pim-setup.sh]: environment var ?

File: .circleci/config.yml (hunk 1 above)
Comment (jmleroux) [reply .circleci/config.yml]: why?

Comment (anaelChardan) [reply .circleci/config.yml]: Why doing this? for blocking in case of error?

File: bin/docker/pim-setup.sh (hunk 3 above)
Comment (anaelChardan) [reply bin/docker/pim-setup.sh]: still not merged

File: .circleci/config.yml (hunk 1 above)
Comment (ahocquard) [reply .circleci/config.yml]: yes, it's the goal of this build actually ;)

Comment (anaelChardan) [reply .circleci/config.yml]: Ok, fine so ;) Otherwise I didn't understood :D

**Pull request merged**